docs/source/conf.py

```diff
@@ -27,11 +27,12 @@ class DocBuildError(Exception):
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 root_path = os.path.abspath(os.path.join("..", ".."))
 sys.path.insert(0, root_path)
-import commands
-_sphinx_apidoc = commands.getoutput('which sphinx-apidoc').strip()
+from avocado.utils import process
+_sphinx_apidoc = process.system_output('which sphinx-apidoc', shell=True).strip()
 _output_dir = os.path.join(root_path, 'docs', 'source', 'api')
 _api_dir = os.path.join(root_path, 'virttest')
-_status, _output = commands.getstatusoutput("%s -o %s %s" % (_sphinx_apidoc, _output_dir, _api_dir))
+_cmdres = process.run("%s -o %s %s" % (_sphinx_apidoc, _output_dir, _api_dir), shell=True)
```

Comment:
A trailing newline is stripped from the output for commands.getstatusoutput()method,so if replaced by process.run, it also need deal with trailing newline issue.

Reply:
@chunfuwen Done